forage-android/src/main/java/com/joinforage/forage/android/network/model/ForageApiResponse.kt

```diff
@@ -6,9 +6,23 @@ internal val UnknownErrorApiResponse = ForageApiResponse.Failure.fromError(
     ForageError(500, "unknown_server_error", "Unknown Server Error")
 )
 
+/**
+ * A model that represents the possible types of responses from the Forage API.
+ *
+ */
 sealed class ForageApiResponse<out T> {
+    /**
+     * A model that represents a success response from the API.
+     *
```

Comment:
can remove newline

Reply:
Can we add some additional context, like "In most cases `data` is a string representation of a JSON object from the Forage API"

 and consider quoting an example (either when we introduce `@sample` or by simply mentioning:

`if (response is ForageApiResponse.Success) { response.data // { "ref": "abcde123", ... }`